scripts/upload-to-pypi.sh

```diff
@@ -11,6 +11,6 @@ VER=$1
 set -e
 
 echo "==> Packaging codalab"
-python3 setup.py sdist
+python3 setup.py bdist_wheel
```

Comment:
Can you ensure we publish _both_ eggs and wheels? It's good to publish source distributions (eggs) as well, see this guide: https://packaging.python.org/guides/distributing-packages-using-setuptools/#packaging-your-project

Reply:
Sure. I just pass both argument to it.